docs/configuration/environment.md

```diff
@@ -429,6 +429,12 @@ Equivalent to the `--torch-backend` command-line argument (e.g., `cpu`, `cu126`,
 Used ephemeral environments like CI to install uv to a specific path while preventing
 the installer from modifying shell profiles or environment variables.
 
+### `UV_USE_ACTIVE_ENVIRONMENT`
+
+Equivalent to the `--active` command line argument. If set, uv will look at `VIRTUAL_ENV`
+and use the active virtual environment, if any. If `VIRTUAL_ENV` is not defined, this has
```

Comment:
Should we explain a bit more of the nuance here? Like... that this is the default behavior outside of projects and scripts with inline metadata?